benchopt/cli/helpers.py

```diff
@@ -40,6 +44,141 @@ def clean(benchmark, token=None, filename=None):
     rm_folder(cache_folder)
 
 
+def print_info(cls_list, env_name=None):
+    """Print information for each element of input listed
+
+    Parameters
+    ----------
+    cls_list : list
+        List of objects (solvers or datasets) to print info from.
+    env_name : str | None
+        Name of conda environment where to check for object availability.
+        If None or False, no check is made.
+    """
+
+    print("-" * 10)
+
+    for cls in cls_list:
+        print(f"## {cls.name}")
+        # doc
+        if hasattr(cls, '__doc__') and cls.__doc__:
+            print(f"> doc: {cls.__doc__}")
+        # parameters
+        if hasattr(cls, 'parameters') and cls.parameters:
+            print("> parameters:")
+            for param, value in cls.parameters.items():
+                values = ', '.join(map(str, value))
+                print(f"    {param}: {values}")
+        # install command
+        if hasattr(cls, 'requirements') and cls.requirements:
+            print("> requirements:")
+            packages = cls.requirements
+            pip_packages = [pkg[4:] for pkg in packages
+                                if pkg.startswith('pip:')]
+            conda_packages = [pkg for pkg in packages
+                                if not pkg.startswith('pip:')]
+            if len(conda_packages) > 0:
+                print("    conda install -c conda-forge " +
+                        f"{' '.join(conda_packages)}")
+            if len(pip_packages) > 0:
+                print(f"    pip install {' '.join(pip_packages)}")
+        else:
+            print("> no dependencies")
+        # availability in env (if relevant)
+        if env_name is not None:
+            # check for dependency avaulability
+            if cls.is_installed(env_name):
+                print(colorify(u'\u2713', GREEN), end='', flush=True)
```

Comment:
Can you put the cross as a constant?

Reply:
Yes, is there a specific file where I should put this ? (in `benchopt.utils` I guess?) or I should create one ?

Reply:
I would put it in `colorify.py` as this will be renamed `terminal_output.py` in #265  ;)

Reply:
ok thanks

Reply:
done